src/Avatar/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { stringToColor } from '../util/colorMapping';
+import { Props } from './props';
+import { styles } from './styles';
+
+export const Avatar = ({
+  avatarUrl = undefined,
+  username = undefined,
+  isOnline = undefined,
+  showOnline = false,
+  style = {},
+  onClick = () => {}
+}: Props) => {
+
+  const text = username ? username.substring(0, 2).toUpperCase() : ''
+  const color = stringToColor(username)
+
+  return (
+    <div style={{ width: '48px', height: '48px' }} onClick={onClick}>
+      <div style={{ height: '0px' }}>
+        <div
+          className='ce-avatar'
+          style={{
+            ...styles.avatar,
+            ...style,
```

Comment:
Put custom `style` prop as the last one so it can over-ride the previous two. This one always needs to go last now 👍